common/src/main/java/org/dash/wallet/common/Configuration.java

```diff
@@ -165,6 +168,11 @@ public boolean remindBackup() {
     public boolean remindBackupSeed() {
         return prefs.getBoolean(PREFS_KEY_REMIND_BACKUP_SEED, true);
     }
+    public boolean lastDismissedReminderMoreThan24hAgo() {
+        long now = System.currentTimeMillis();
+        long lastReminder = prefs.getLong(PREFS_KEY_BACKUP_SEED_LAST_DISMISSED_REMINDER, now);
+        return now - lastReminder > 86400000;
```

Comment:
Hi @denlb ,
please replace this magic number `86400000` with something more readable, like `TimeUnit.HOURS.toMillis(24)`.

Reply:
Done